doc/releases/release-notes-2.2.rst

```diff
@@ -5,10 +5,11 @@
 Zephyr 2.2.0 (Working Draft)
 ############################
 
-We are pleased to announce the release of Zephyr kernel version 2.2.0.
+We are pleased to announce the release of Zephyr RTOS version 2.2.0.
 
 Major enhancements with this release include:
 
+* We added support for 64-bit ARMv8-A Architecture (Experimental).
```

Comment:
Please, do not use the word `experimental`. It is not an experiment and really detrimental to the work done. We can use `preliminary support` or `initial support`.

Reply:
"initial" is fine (actually required, so I will add it)

"experimental" is hat we have been adding in similar cases - does not mean "it is not working" - it gives the impressions that something is not yet to be fully trusted; we don't make promises.

Reply:
ok

Reply:
And thanks once more @carlocaione for adding the support for v8-A :)

Reply:
np ;) kudos to all the contributors.